test/cut/test_custom_attrs.py

```diff
@@ -0,0 +1,19 @@
+from tempfile import NamedTemporaryFile
+
+import numpy as np
+
+from lhotse import LilcomHdf5Writer, MonoCut
+from lhotse.array import Array
+
+
+def test_cut_load_array():
+    array = np.arange(20).astype(np.float32)
+    with NamedTemporaryFile(suffix=".h5") as f, LilcomHdf5Writer(f.name) as writer:
+        manifest = Array.store(key="utt1", value=array, writer=writer)
```

Comment:
I was a little confused here by the name "manifest" as I had assumed "manifest" always refers to a collection of metadata, but I suppose it refers to an item of metadata here?  I wonder if those should be given different names...

Reply:
This is correct — calling Array.store returns a manifest for the stored data. Not sure how to balance informativeness and verbosity — Array.store_and_create_manifest is a mouthful. Do you have any suggestions?

Reply:
I changed it to `manifest = writer.store_array(...)` in the other PR, maybe this is more helpful?